definitions/contested/json/CaseEventToFields/CaseEventToFields-respSolJourney-nonprod.json

```diff
@@ -18,7 +18,7 @@
     "CaseEventID": "FR_amendApplication",
     "CaseFieldID": "RespondentOrganisationPolicy",
     "PageFieldDisplayOrder": 3,
-    "DisplayContext": "OPTIONAL",
+    "DisplayContext": "COMPLEX",
```

Comment:
needs to add this event FR_amendApplication to CaseEventToComplexType, alike FR_solicitorCreate